tests/functional/features/login.js

```diff
@@ -0,0 +1,63 @@
+import { genEmail, getLocation, pages, SITE } from '../helpers'
+import { HomePage } from '../page-objects/home-page'
+import { LoginRegisterPage } from '../page-objects/login-register-page'
+import { DashboardPage } from '../page-objects/dashboard-page'
+
+const { login, signUp, signUpNoDisclaimer } = LoginRegisterPage
+const password = 'Foo@1234foo'
+
+fixture `User login`
+  .beforeEach(async t => {
```

Comment:
Instead of using beforeEach can this be changed to a function that each test needs to call?

Reply:
See the quote excerpt that Kenny put in his original review comments (extract below):

> Avoid references to shared state — especially shared mutable state between unit tests. Instead, use factory functions (as mentioned above).

> I’ve seen too many suites using beforeEach() and afterEach() accidentally mutate shared state between tests, only to cause problems when the order of test execution gets rearranged, or when tests are run in parallel in order to save time.